### PR TITLE
Change mentions of npm Enterprise to npm On-Site

### LINKED ISSUE #374: npm Enterprise -> npm On-site

### PR DESCRIPTION
**Pull request by lewiscowper**
# Long version

This PR goes some of the way to solving #374, however it is important to note before this is merged that there are a number of references to https://www.npmjs.com/package/npme which references the "Enterprise" name again, and will presumably be replaced by https://www.npmjs.com/package/npmo.

However npmo's documentation ask the user to install with sudo and globally, so the instructions in [installation.md](https://github.com/lewiscowper/docs/blob/enterprise-to-on-site/content/on-site/installation.md) will be incorrect, and be more than just a straight swap.

I don't mind looking deeper into it to fix the documentation around the `npme` => `npmo` swap too, just thought I'd put this up in case it's the intended fix for #374.

# Short version

This PR changes `npm Enterprise` to `npm On-site` to fix #374.
This PR does not change the docs surrounding `npme` vs `npmo` -- Happy to help with that, but it'd require more research.

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
omg @lewiscowper :sparkling_heart:

**Comment by ashleygwilliams** [comment]
this is such a great start. @seldo is this how you imagined the switch would look (thinking URL names and such)? still have a few things to change here, but well on our way, thanks so much @lewiscowper !!

**Comment by lewiscowper** [comment]
Also I stuck with "On-site" as capitalised in #374, but I did notice that elsewhere (specifically https://www.npmjs.com/onsite) On-Site is used. I can change them and rebase the commit if that's the preferred name.

**Comment by seldo** [comment]
The canonical, marketing approved name is as found on https://www.npmjs.com/onsite, i.e. "npm On-Site", and the url is `/onsite` rather than `/on-site`. This is awesome!

**Comment by lewiscowper** [comment]
I'll make the change tonight, rebase and push with those changes tomorrow morning. :+1:

**Comment by nexdrew** [comment]
Not sure why this was never merged, but it seems to have been superseded by #445.

**Comment by ashleygwilliams** [comment]
closing. sorry i missed this :(
